src/engines/Cesium/hooks/useLayerDragDrop.ts

```diff
@@ -0,0 +1,77 @@
+import { Cartesian3, Entity, Viewer } from "cesium";
+import CesiumDnD, { Context } from "cesium-dnd";
+import { RefObject, useCallback, useEffect, useRef } from "react";
+import { CesiumComponentRef } from "resium";
+
+import { LatLng } from "../../../utils";
+import { isDraggable, isSelectable } from "../common";
+import { getTag } from "../Feature";
+import { convertCartesian3ToPosition } from "../utils/utils";
+
+export default ({
```

Comment:
BTW is this still necessary?

Reply:
Not necessary in my mind...but i'm not 100% sure.